timemachine/cpp/src/verlet_integrator.cu

```diff
@@ -0,0 +1,107 @@
+
```

Comment:
I would be fine putting them in the same file or otherwise. @maxentile could probably comment on the desire for additional variants.

Reply:
I don't foresee a situation where it would be confusing to use terms "Verlet" and "velocity Verlet" interchangeably in this context.

Main variation I can currently foresee is to potentially simplify the staggering-by-half-a-timestep complication in the trajectory returned by multiple steps (e.g. https://github.com/proteneer/timemachine/pull/783), but that would still be velocity verlet, just with a slightly different interface.

Reply:
Makes sense. Is it worth then updating the filename to match the class it defines? (nit)

Reply:
Inclined to leave it, but only because the longer file name doesn't provide value when the class name is qualified and we refer to BAOAB as langevin even though there are many variants of langevin.